test/lint/lint-whitespace.sh

```diff
@@ -22,14 +22,6 @@ while getopts "?" opt; do
   esac
 done
 
-if [ -z "${TRAVIS_COMMIT_RANGE}" ]; then
```

Comment:
Please do not remove this part, change the `TRAVIS_COMMIT_RANGE` to `COMMIT_RANGE` ibstead.